src/psyclone/tests/psyir/frontend/where_handler_test.py

```diff
@@ -116,8 +117,8 @@ def test_where_array_notation_rank(parser):
     ''' Test that the _array_notation_rank() utility raises the expected
     errors when passed an unsupported Array object.
     '''
-    from psyclone.psyir.symbols import DataType
-    my_array = Array("my_array", None)
+    from psyclone.psyir.symbols import DataType, DataSymbol
```

Comment:
pylint complains that `parser` in unused in this test. Please use `@pytest.mark.usefixtures`

Reply:
Changed and changed all others (that were required) in the file too.